src/helpers.ts

```diff
@@ -27,6 +33,32 @@ const variable = (name: any): string[] => {
   return values;
 };
 
+function formateDate(name: string, value: string): string {
```

Comment:
```suggestion
function formatDate(name: string, value: string): string {
```

Reply:
oops..  it is a typo. thanks  😄

I resolved.

Reply:
I also have fixed linter issues in CI report. thank you~